packages/venia-ui/lib/components/FilterModal/filterModal.js

```diff
@@ -66,9 +69,49 @@ const FilterModal = props => {
         </div>
     ) : null;
 
+    function changeFocusOnModal(event) {
```

Comment:
Both of these should be moved to `useFilterModal` and follow standard conventions.

`const handleFocusOnOpen = useCallback(event => {...})`